app/models/slot.rb

```diff
@@ -128,4 +129,14 @@ def add_transfert_error(transfert, attribute, place_name)
                I18n.t("activerecord.errors.models.slot.attributes.date.#{attribute}", date: transfert.date.to_fs,
                                                                                       place_name:))
   end
+
+  def in_the_future
```

Comment:
Ahah t'es vraiment pas chaud de faire 2 méthodes pour que ce soit cohérent avec leur nom ?! 😉

Reply:
tu veux in `in_the_future` et `in_the_near_future`?
ca semble pas hyper necessaire non

Reply:
Aussi, la date n'est pas bloquée en front sur les slots

Reply:
Ce qui est chelou c'est que dans la méthode qui vérifie que c'est dans le futur on vérifie que c'est pas trop loin dans le futur. Le nom ne reflète pas ce que la méthode fait. Du coup oui, je ferais un truc comme ça en l'appelant sans doute plutôt "not_too_far_from_now" pour marquer la différence.